account_move_base_import/models/account_journal.py

```diff
@@ -0,0 +1,366 @@
+# Copyright 2011-2016 Akretion
+# Copyright 2011-2019 Camptocamp SA
+# Copyright 2013 Savoir-faire Linux
+# Copyright 2014 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+import os
+import sys
+import traceback
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+from ..parser.parser import new_move_parser
+
+
+class AccountJournal(models.Model):
+    _name = "account.journal"
+    _inherit = ["account.journal", "mail.thread"]
```

Comment:
It's seems unnecessary inherit "mail.thread" here because this are already made in main module https://github.com/OCA/OCB/blob/13.0/addons/account/models/account.py#L695